File: test/fixtures/javascript-project/utils/validators.js
const EMAIL_REGEX = /^[\w.-]+@[\w.-]+\.\w+$/;

function validateEmail(email) {
  return EMAIL_REGEX.test(email);
}

function validateName(name) {
  return typeof name === 'string' && name.trim().length >= 2;
}

function sanitizeInput(input) {
  if (typeof input !== 'string') return '';
  return input.trim().replace(/[<>]/g, '');
}

module.exports = { validateEmail, validateName, sanitizeInput };

File: test/fixtures/go-project/utils/validators.go
package utils

import "regexp"

var emailRegex = regexp.MustCompile(`^[\w.-]+@[\w.-]+\.\w+$`)

func ValidateEmail(email string) bool {
	return emailRegex.MatchString(email)
}

func ValidateName(name string) bool {
	return len(name) >= 2
}
